tests/test_build.py

```diff
@@ -182,7 +182,7 @@ def test_create_info_files_json_no_inodes(testing_workdir, testing_metadata):
 
 
 def test_rewrite_output(testing_workdir, testing_config, capsys):
-    api.build(os.path.join(metadata_dir, "rewrite_env"), config=testing_config)
+    api.build(os.path.join(metadata_dir, "_rewrite_env"), config=testing_config)
```

Comment:
I moved this to _rewrite_env because it is a test that we want to manually run.  Things starting with _ are not run automatically with https://github.com/conda/conda-build/blob/master/tests/test_api_build.py#L82-L94